### PR TITLE
Enable OpInfo CUDA tests

### PR DESCRIPTION
**Pull request by SherlockNoMad**
Enable OpInfo CUDA tests in CI
Updated xfail list for CUDA. 
Turned on FAIL_ON_SUCCESS.

### CODE REVIEW

File: test/test_torchinductor_opinfo.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
 TestExpect = Enum("TestExpect", ("SUCCESS", "XFAILURE", "SKIP"))
 
 COLLECT_EXPECT = os.getenv("PYTORCH_COLLECT_EXPECT", "0") == "1"
-FAIL_ON_SUCCESS = os.getenv("PYTORCH_FAIL_ON_SUCCESS", "0") == "1"
+FAIL_ON_SUCCESS = os.getenv("PYTORCH_FAIL_ON_SUCCESS", "1") == "1"
```
Comment (SherlockNoMad) [inline test/test_torchinductor_opinfo.py:43]: I am turning on FAIL_ON_SUCCESS...
If tests are flaky, we can revert

Comment (desertfire) [reply test/test_torchinductor_opinfo.py:43]: The new CI results seem to suggest `__rdiv__` is flaky? You can move it to the skip list first.

Comment (SherlockNoMad) [reply test/test_torchinductor_opinfo.py:43]: yes, done.

**Review by desertfire — APPROVED** [review]
Land after CI passes

**Pull request merged**